lib/writeLog.js

```
@@ -42,7 +42,8 @@ function writeLog(commits, options, done) {
   if (!writer.header()) {
     return done('No version specified');
   }
-  writer.section('Bug Fixes', sections.fix);
+  writer.section('Minor fixes', sections.fix);
+  writer.section('Bug Fixes', sections.bug);
```

This requires that we use our fork of this utility, and anyone who uses the stock version from npm will generate the Bug fixes section containing the minor fixes.  I don't know of a way to make the stock version not fail silently, so we'll have to consider how to insure that this version gets installed.  Currently, I'm using the stock version because we hadn't made any significant changes to this fork.